### DISCUSSION TITLE
How to specify --endpoint-url and --no verify-ssl

### DISCUSSION THREAD

**Discussion by dotnetwizard** [aws/aws-sdk-net#2616] [Q&A] [answered]
How do I specify the cli equivalent of --endpoint-url and --no verify-ssl ? 

Thanks in advance.

**Comment by ashishdhingra** [comment, marked as answer]
@dotnetwizard Good afternoon. All service clients (e.g. `AmazonS3Client`) support an overloaded constructor that allow you to pass service specific `ClientConfig` object (e.g. `AmazonS3ClientConfig`). The base class [ClientConfig](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/Runtime/TClientConfig.html) supports `ServiceURL` property which maps to CLI's `--endpoint-url`. The `--no-verify-ssl` option in AWS CLI bypasses (overrides) the default behavior of verifying SSL certificates. Unfortunately, per https://github.com/aws/aws-sdk-net/issues/686#issuecomment-313287324, AWS SDK for .NET does not support server side SSL validation.

Thanks,
Ashish
